src/components/List/List.jsx

```diff
@@ -1,25 +1,26 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
 import { getPosts } from '../../selectors/Selectors';
+import Post from '../Post/Post';
 
-import Post from '../../components/Post/Post';
+const List = () => {
 
-export default function List() {
-  const posts = useSelector(getPosts);
+  const post = useSelector(getPosts);
 
-  const postElements = posts
-    .map((post, i) => (
-      <li key={post.index}>
-        <Post {...post}/>
-      </li>
-    ));
+  const postElements = post.map(post => (
+    <li key={post.index}>
+      <Post {...post}/>
```

Comment:
I don't remember but I think you need to explicitly pass down the index. In Post you're calling your prop, Index so 
```
<Post Index={i} {...post} />
```